test/utils.sh

```diff
@@ -28,48 +56,19 @@ announce() {
 }
 
 has_namespace() {
-  if $cli get namespace  "$1" > /dev/null; then
+  #We dont need retry because some of the scenarios waiting for false
+  if $cli_no_timeout get namespace  "$1" > /dev/null; then
     true
   else
     false
   fi
 }
 
-wait_for_it() {
-  local timeout=$1
-  local spacer=2
-  shift
-
-  if ! [ $timeout = '-1' ]; then
-    local times_to_run=$((timeout / spacer))
-
-    echo "Waiting for '$@' up to $timeout s"
-    for i in $(seq $times_to_run); do
-      eval $@ > /dev/null && echo 'Success!' && return 0
-      echo -n .
-      sleep $spacer
-    done
-
-    # Last run evaluated. If this fails we return an error exit code to caller
-    eval $@
-  else
-    echo "Waiting for '$@' forever"
-
-    while ! eval $@ > /dev/null; do
-      echo -n .
-      sleep $spacer
-    done
-  echo 'Success!'
-  fi
-}
-
-
 set_namespace() {
   if [[ $# != 1 ]]; then
     printf "Error in %s/%s - expecting 1 arg.\n" "$(pwd)" $0
     exit 1
   fi
-
```

Comment:
why was this line removed?

Reply:
didnt see a reason for empty line after if(other function doesnt contain empty line after if)

Reply:
I think the extra line looks better but if this pattern fits the rest of the file then it;s ok.